etl-federation/existing-prometheus/existing-prometheus/README.md

```diff
@@ -0,0 +1,71 @@
+# Overview
+
+These are the steps to enable ETL federation with Kubecost.
+
+This is an efficient method to implement multi-cluster Kubecost while using existing prometheus installations. Contact us for help customizing settings.
```

Comment:
I recommend to add a hyperlink here:
Example:

[Contact us](https://www.kubecost.com/contact) for help customizing settings.